backend/graphql/types/testType.ts

```diff
@@ -109,7 +127,7 @@ const testType = gql`
   extend type Mutation {
     createTest(test: TestRequestDTO!): TestResponseDTO!
     updateTest(id: ID!, test: TestRequestDTO!): TestResponseDTO!
-    deleteTestById(id: ID!): ID
+    deleteTestById(id: ID!): ID!
```

Comment:
curious to know if this was added because you ran into an error? but i do think it should have one

if you ran into an error, we might want to fix any delete functions with a missing `!` in the other files

Reply:
I don't think it got errors. I was just removing `!` to troubleshoot another error (because the existing tests have null for the added fields and was erroring and I fixed it by changing the data in the database). I guess when I added them back I accidentally added an extra one

Reply:
wait so there shouldn't be an extra `!`? (remove it pls if not)